addons/website_sale/security/website_sale.xml

```diff
@@ -44,4 +44,17 @@
     <record id="sales_team.group_sale_manager" model="res.groups">
         <field name="implied_ids" eval="[(4, ref('website.group_website_publisher'))]"/>
     </record>
+
+    <record id="base.group_public" model="res.groups">
```

Comment:
what is purpose to implied 'group_delivery_invoice_address' into public, portal group ?